tools/build_utils.py

```diff
@@ -190,17 +220,23 @@ def setup_venv(venv_dir):
     load_venv(venv_dir)
 
     print("PIP location")
-    process = subprocess.Popen(shlex.split('which pip'))
-    so, se = process.communicate()
+    if (platform.system() == 'Windows'):
+        call(['where', 'pip'])
+    else:
+        process = subprocess.Popen(shlex.split('which pip'))
+        so, se = process.communicate()
 
     # Install the pip packages
-    command_executor(["pip3", "install", "-U", "pip"])
+    if (platform.system() == "Windows"):
+        command_executor(["python -m pip", "install", "-U", "pip"])
+    else:
+        command_executor(["pip", "install", "-U", "pip"])
     package_list = [
         "pip3",
         "install",
         "psutil",
         "six>=1.12.0",
-        "numpy>=1.16.0,<1.19.0",
+        # "numpy>=1.19.5",
```

Comment:
Can you please remove the comment?

Reply:
TF on Windows needs a higher version of numpy, handled conditionally